sympy/printing/ccode.py

```diff
@@ -666,6 +671,12 @@ def ccode(expr, assign_to=None, standard='c99', **settings):
         Setting contract=False will not generate loops, instead the user is
         responsible to provide values for the indices in the code.
         [default=True].
+    expand_int_pow : callable, ``f(base, exp) -> bool``, optional
+        Predicate for whether to expand ``pow`` of integer power into
+        multiplications (instead of calling ``pow``). Note that most compilers
+        will do this for you when passed the correct optimization flags. If
+        use this predicate you probably want to ensure that ``base.is_symbol``
+        and that ``exp`` is not too large.
```

Comment:
Would it be worth allowing simply "True" and "False" too? For simple cases, creating a lambda func seems like unnecessary overhead/

Reply:
Sure, what would a reasonable default be? True => `lambda b, e: b.is_symbol e < 42`?

Reply:
Sure :)

Reply:
I went with allowing it to be an integer instead, what do you think: helpful or confusing?

Reply:
I like it.